src/Adapters/EventSource.Netstandard13/EventSourceTelemetryModule.cs

```diff
@@ -65,6 +72,11 @@ public EventSourceTelemetryModule(OnEventWrittenHandler onEventWrittenHandler)
         public IList<EventSourceListeningRequest> Sources { get; private set; }
 
         /// <summary>
+        /// Gets the list of the EventSource disalbing rules in cases there's known event source that is causing issues.
```

Comment:
disabling requests

(typo + it is a collection of requests)

Reply:
Corrected in the next iteration. Thanks!